.github/workflows/gcp_trustedboot_tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#set -Eeuo pipefail
+
+# Name of Image to test
+cname="${@: -1}"
+
+configFile="gcp_test_config.yaml"
+containerName="ghcr.io/gardenlinux/gardenlinux/integration-test:today"
```

Comment:
Please update according to latest changes in `gcp_tets.sh`.
`containerName` changed.